src/Scheduler.py

```diff
@@ -41,11 +52,24 @@ def cancel_scheduled_msg(self, scheduled_time_str, contact):
         else:
             print("Failed to cancel msg to contact= {0} at time= {1}".format(contact, scheduled_time_str))
 
+    def init_barcode(self, chrome_path_driver):
```

Comment:
Shouldn't init_barcode be in MsgSender?
The scheduler is only responsible for calling MsgSender when needed.

Reply:
Right